tools/tests.xml

```diff
@@ -3,6 +3,7 @@
   <test name="host" type="unit" binary="Conhost.Unit.Tests.dll" />
   <test name="textBuffer" type="unit" binary="TextBuffer.Unit.Tests.dll" />
   <test name="terminalCore" type="unit" binary="Terminal.Core.Unit.Tests.dll" />
+  <test name="terminalApp" type="unit" binary="UnitTests_TerminalApp\Terminal.App.Unit.Tests.dll" />

```

Comment:
Why is this one not binplaced where the others are? Did you confirm the CI can see it in the subdirectory?

Reply:
Because cppwinrt reasons. All cppwinrt projects get binplaced into their own directory, so that they don't get re-included into other projects. I tried last year to get them to all binplace into `bin/` and it was horrifying. The tests definitely _do_ run in CI though, so that's good!